### ISSUE TITLE
We try to import the analytic views on load-from-h2, causing a "ERROR: duplicate key value violates unique constraint "idx_xxx"

### ISSUE THREAD

**Issue by paoliniluis** [metabase/metabase#46110]
### Describe the bug

If you connect Metabase to an app db and then try to import an h2 dump, then the views we create for analytics will already be there, but the process will try to also load the analytics tables that are in the dump, causing an error

### To Reproduce

1) initialize metabase
2) dump it to h2
3) then initialize another metabase
4) try to load-from-h2 the previous dump

### Expected behavior

We should not dump the analytics tables in the process, as those tables should always be in the destination

### Logs

NA

### Information about your Metabase installation

```JSON
v49+
```

### Severity

P2

### Additional context

workaround: initialize the new MB environment with MB_LOAD_ANALYTICS_CONTENT=false

**Comment by perivamsi** [comment]
This should be a P1 since importing from h2 is completely broken.

**Comment by paoliniluis** [comment]
@perivamsi agree, the workaround that I posted doesn't work

**Comment by johnswanson** [comment]
I was able to reproduce on v49 was unable to repro on v50 or `master`. I _think_ [this](https://github.com/metabase/metabase/pull/40907/files#diff-65fdb622c6760fde1867daf5516614006d0c0fb106cf63f5718cd31bde2ebbefR202) may have fixed the issue, although I'm a little confused that @paoliniluis was able to repro on July 24 (this PR was merged Apr 3). @paoliniluis were you using a specific version or v50+/`master`?

If it's fixed as of v50 I think we could probably close this since it has a workaround and v49 we're releasing v52 soon.
